Package.swift

```diff
@@ -143,5 +39,359 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-collections.git", from: "1.0.2"),
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
     ],
-    targets: targets
+    targets: [
+        // MARK: Targets
```

Comment:
```suggestion
        // MARK: - Targets
```
Makes use of Xcode's formatting for splitting up stuff